english/vote-the-president/trump/trump.md

```diff
@@ -1,3 +1,5 @@
 You made America great again. Now its time to build a wall. Call 987654321 to contact Bob the builder.
 
-It's gonna be huuuuge!
\ No newline at end of file
+It's gonna be huuuuge!
+
+[Uh-Oh!](../deep-meditation/deep_meditation.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.